PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,6 @@
 - [ ] A11y testing - Have you run pa11y against the component and fixed/verified all errors, warnings and notices?
 
 ## Have the following been considered/are they needed?
-
 - [ ] Tests?
 - [ ] Docs?
+- [ ] Have you had a conversation with the right people about the implications of this feature?
```

Comment:
I’d suggest the less formal version from the PR description:

> Have you spoken to the right people about the implications of this feature?

Or, given that the other items are all short reminders, not full checklist items, maybe just:

> Spoken to the right people?